app/assets/stylesheets/govuk_publishing_components/components/govspeak/_attachment.scss

```diff
@@ -70,6 +70,11 @@
         @include govuk-font($size: 14);
       }
 
+      .url {
+        word-break: break-word;
+        word-wrap: break-word;
```

Comment:
Is it worth documenting why we're setting both `word-break` and `word-wrap`? Or does this exist as prior art somewhere? I don't think I've seen the two properties used together before.

Reply:
Sure, I can add more details in the commit message. `word-wrap` is meant to be the non-standard Microsoft implementation of `overflow-wrap` that I was trying to leverage since `word-break` is not supported.